src/components/interaction-designer/toolbar/ErrorNotifications.vue

```diff
@@ -123,9 +123,17 @@ export class ErrorNotifications extends mixins(Routes, Lang) {
 
   get invalidBlocksInActiveFlow() {
     return filter(this.activeFlow?.blocks, (block) => {
-      // The block is invalid on IBlock OR it's invalid on corresponding IResource
-      return get(this.blockValidationStatuses, `block/${block.uuid}`) || get(this.resourceValidationStatusesForActiveFlow, `resource/${block.config.prompt}`)
-    });
+      const hasBlockValidationError = has(this.blockValidationStatuses, `block/${block.uuid}`)
+      const hasResourceValidationError = has(this.resourceValidationStatusesForActiveFlow, `resource/${block.config.prompt}`)
+      let hasChoiceValidationError = false
+
+      if (block.config?.choices !== undefined) {
```

Comment:
Could you explain a bit why we need `hasChoiceValidationError` ?

Reply:
No particular reason apart from I needed some distinction between content-resources and choice-option-resources during the development.

I removed it in the current version.